sql/sql_expression_operations.cc

```diff
@@ -1171,8 +1171,12 @@ static CellValue binaryModulus(const CellValue & la, const CellValue & ra)
 struct BinaryPlusOp {
     static CellValue apply(const CellValue & l, const CellValue & r)
     {
-        if (l.empty() || r.empty())
+        if (l.empty()) {
             return l;
+        }
+        if (r.empty()) {
+            return r;
+        }
```

Comment:
@jeremybarnes Any timestamp logic to take into account here? Should it return the max ts between l and r?